pkgs/applications/networking/iptraf-ng/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.4";
+  name = "iptraf-ng-${version}";
+
+  src = fetchurl {
+    url = "https://fedorahosted.org/releases/i/p/iptraf-ng/${name}.tar.gz";
+    sha256 = "02gb8z9h2s6s1ybyikywz7jgb1mafdx88hijfasv3khcgkq0q53r";
+  };
+
+  buildInputs = [ ncurses ];
+
+  configurePhase = ''
```

Comment:
Because it won't build?

On 2 October 2015 10:19:33 BST, lethalman <notifications@github.com> wrote:
>> @@ -0,0 +1,44 @@
>> +{ stdenv, fetchurl, ncurses }:
>> +
>> +stdenv.mkDerivation rec {
>> +  version = "1.1.4";
>> +  name = "iptraf-ng-${version}";
>> +
>> +  src = fetchurl {
>> +    url =
>"https://fedorahosted.org/releases/i/p/iptraf-ng/${name}.tar.gz";
>> +    sha256 = "02gb8z9h2s6s1ybyikywz7jgb1mafdx88hijfasv3khcgkq0q53r";
>> +  };
>> +
>> +  buildInputs = [ ncurses ];
>> +
>> +  configurePhase = ''
>
>Why isn't the default configure enough?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/NixOS/nixpkgs/pull/10173/files#r41003905